examples/aggregator/gcp/pod.yaml

```diff
@@ -10,11 +10,26 @@ spec:
       image: gcr.io/oak-ci/oak-aggregator:latest
       ports:
         - containerPort: 8080
+      volumeMounts:
+        - name: tls-secret
+          mountPath: '/etc/oak-secrets'
+          readOnly: true
+      args:
+        [
+          '--application=config.bin',
+          '--ca_cert=ca.pem',
+          '--cert_chain=docker.pem',
```

Comment:
We can store all TLS related files in the `/etc/oak-secrets/` (just like `.ssh`), so they will not be scattered in different places

Reply:
I don't think it is possible to mount a secret to an existing directory that contains the other files. It would be possible to provision all certificate files (even the non-secret ones) using k8s secrets as @rbehjati suggested.